### ISSUE TITLE
Команда noskills

### ISSUE THREAD

**Issue by CASPERX69XSOFT** [FWGS/mainui_cpp#71]
Ранее эта команда убирала боковые подсказки в меню, сейчас это не работает.

Команда так же убирала выбор сложности, желательно реализовать отдельно команду для отключения подсказок, чтоб не убирать сразу 2-е функции

**Comment by Vladislav4KZ** [comment]
Зачем надо было закрывать прошлый issue? Мог бы просто переименовать тему.

**Comment by nekonomicon** [comment]
> Мог бы просто переименовать тему.
Нет, ему же перенести нужно было, в любом случае перенести может владелец репозиториев.

> Команда noskills
Параметр в liblist.gam в WON-версии HL или именно команда?

**Comment by a1batross** [comment]
noskills никогда не убирал подсказки.

**Comment by a1batross** [comment]
https://github.com/a1batross/Xash3D_original/blob/59943f431999ea8960cab2ec236ba8f755d0a829/mainui/utils.cpp#L2174

Беру свои слова обратно...

**Comment by SNMetamorph** [comment]
Ага, он и в ванильном ксаше убирал их
